### PR TITLE
Link customers to TFE_PARALLELISM knowledge base rather than ask them to cut tickets

### PR DESCRIPTION
**Pull request by drudgesentinel**
It's better to emphasize that thorough understanding of the underlying parallelism is prerequisite, rather than asking telling customers to ask support (as support has no insight into customer resources, nor do we have a way to predict how parallelism adjustments will impact their run completion times)

Understanding API usage thoroughly and setting parallelism is an architecture consideration, which support typically does not field

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Review by mgarrell777 — APPROVED** [review]
@goldendemise Thanks very much for your effort on this PR. Looks good to me.

File: website/docs/cloud-docs/workspaces/variables/index.mdx [hunk 1]
```diff
@@ -29,7 +29,7 @@ Environment variables can store provider credentials and other data. Refer to yo
 
 You can use the `TFE_PARALLELISM` environment variable when your infrastructure providers produce errors on concurrent operations or use non-standard rate limiting. The `TFE_PARALLELISM` variable sets the  `-parallelism=<N>` flag for  `terraform plan` and `terraform apply`  ([more about `parallelism`](/terraform/internals/graph#walking-the-graph)). Valid values are between 1 and 256, inclusive, and the default is `10`. HCP Terraform agents do not support `TFE_PARALLELISM`, but you can specify flags as environment variables directly via [`TF_CLI_ARGS_name`](/terraform/cli/config/environment-variables#tf-cli-args). In these cases, use `TF_CLI_ARGS_plan="-parallelism=<N>"` or `TF_CLI_ARGS_apply="-parallelism=<N>"` instead.
 
-!> **Warning:** We recommend talking to HashiCorp support before setting `TFE_PARALLELISM`.
+!> **Warning:** We recommend thorough understanding of [Terraform parallelism](https://support.hashicorp.com/hc/en-us/articles/10348130482451) prior to setting `TFE_PARALLELISM`.
```
Comment (rkoron007) [inline website/docs/cloud-docs/workspaces/variables/index.mdx:32]:
```suggestion
!> **Warning:** We recommend reading and understanding [Terraform parallelism](https://support.hashicorp.com/hc/en-us/articles/10348130482451) prior to setting `TFE_PARALLELISM`. You can also contact HashiCorp support for direct advice. 
```

**Review by rkoron007 — APPROVED** [review]
Added a few tweaks, and approving and merging!

**Comment by drudgesentinel** [comment]
It finally happened :smiling_face_with_tear:

**Pull request merged**